activerecord/lib/active_record/railties/databases.rake

```diff
@@ -158,6 +158,21 @@ db_namespace = namespace :db do
     puts "Current version: #{ActiveRecord::Migrator.current_version}"
   end
 
+  namespace :version do
+    desc 'Prints the current schema version number'
+    task :number => [:environment, :load_config] do
+      puts ActiveRecord::Migrator.current_version
+    end
+
+    desc "Prints the file path of current schema's migration"
+    task :file => [:environment, :load_config] do
+      current_version = ActiveRecord::Migrator.current_version.to_s
+      migration_file = Dir.glob("#{ActiveRecord::Migrator.migrations_path}/*.rb")
```

Comment:
Actually github parses my response in other way,, there is an asterisk ( * ) around interpolation of current_version.

* #{ current_version }